package.nls.json

```diff
@@ -419,6 +420,8 @@
     "AWS.s3.downloadFile.progressTitle": "Downloading {0}...",
     "AWS.s3.downloadFile.saveButton": "Download",
     "AWS.s3.downloadFile.error.general": "Failed to download file {0}",
+    "AWS.s3.promptTime.placeHolder": "Defaults to 15 minutes",
```

Comment:
Rename to `Minutes`. You should just initialize this box with a value of `'15'`

Reply:
So don't have the placeholder?

Reply:
Use a placeholder but have it be just the number.
The title should specify that the time is in minutes.

Reply:
I'll rephrase what I had at first:
Placeholder: "Enter time in minutes"
Value (on create): '15'

Placeholder is what's displayed when the field is empty, so '15' doesn't do much as a placeholder. If you want 15 as the default value, initialize the input box with '15'. The user will only see the placeholder text if they empty out the box, in which case they'll also get the invalid message from the validation function.